packages/ripple-binary-codec/src/types/amount.ts

```diff
@@ -147,7 +147,8 @@ class Amount extends SerializedType {
    */
   toJSON(): AmountObject | string {
     if (this.isNative()) {
-      const bytes = this.bytes
+      const bytes = Buffer.alloc(this.bytes.length)
+      this.bytes.copy(bytes)
```

Comment:
This fixes a minor issue I ran into, where I wasn't expecting `toJSON` to modify `this.bytes`